vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -11,6 +11,29 @@ required_env_vars=(
     "IMAGE_VERSION"
 )
 
+# Higher the replication_inverse, lower is the usage and number of replicas
+set -x
+REPLICATION_INVERSE=1
+feature_set=("fips" "gpu" "arm64" "cvm" "tl" "1804")
+if [ "${OFFER_NAME,,}" != "ubuntu" ]; then
+    # Since Ubuntu is our most used SKU as compared to Windows/Mariner/AzLinux, we dont need the same number of replicas for all.
+    # Starting off with half replicas.
+    REPLICATION_INVERSE=$((REPLICATION_INVERSE * 2))
+fi
+
+if [ "${HYPERV_GENERATION,,}" == "v1" ]; then
+    # Gen2 SKUs are more used as compared to Gen1 SKUs, therefore Gen1 SKUs do not warrant the same number of replicas
+    REPLICATION_INVERSE=$((REPLICATION_INVERSE * 2))
+fi
+
+for feature in "${feature_set[@]}"; do
```

Comment:
I see... I guess we wouldn't have this in context of release scripts

Reply:
it looks like this only actually works for kata/gpu SKUs, could maybe just manually check *"gpu"* and *"kata"* and then move all this over?

granted, I'm leaning this way just to avoid breaking up the change between both repos and to minimize what we include in publishing info - curious what your thoughts are on this

Reply:
this also increases the inverse for arm64, TL, CVM

feature_set=("fips" "gpu" "arm64" "cvm" "tl" "1804")
declared on L17, you actually gave me a good one I missed, kata.

Reply:
oh, I completely missed that lol, thought this was using the value of the FEATURE_FLAGS var from the environment - but yeah in that case should definitely be doable at release time, just something to consider